edx_exams/apps/api/v1/views.py

```diff
@@ -426,18 +426,22 @@ def get(self, request):
         """
         user = request.user
         latest_attempt = get_latest_attempt_for_user(user.id)
+        serialized_attempt = StudentAttemptSerializer(latest_attempt)
 
         # if there is an active attempt in this service, return it
         if latest_attempt is not None:
             latest_attempt = check_if_exam_timed_out(latest_attempt)
             serialized_attempt = StudentAttemptSerializer(latest_attempt)
```

Comment:
Note: I'm keeping the serialized attempt, since the line above (check_if_exam_timed_out) can potentially update the attempt status in which case the serialized attempt will be different from the originally set one.